lib/booking.rb

```diff
@@ -0,0 +1,11 @@
+class Booking
+  include DataMapper::Resource
+
+  property :id,           Serial
+  property :arrival,      Date,     required: true
+  property :departure,    Date,     required: true
+  property :confirmed,    Boolean
+
+  belongs_to :space
+
```

Comment:
Does this need to belong_to :user as well?

Reply:
Yes! I think it does